sympy/printing/mathml.py

```diff
@@ -13,34 +13,89 @@
 from .conventions import split_super_sub, requires_partial
 
 
-class MathMLPrinter(Printer):
-    """Prints an expression to the MathML markup language
+class MathPrinter(Printer):
```

Comment:
I think the name `MathPrinter` is too general and it should include 'mathml', like `MathMLPrinterBase`.